css-animations-2/Overview.bs

```diff
@@ -770,6 +770,8 @@ for an [=animation trigger=] that enters its [=animation trigger/active interval
 
 The behavior of each value is defined in [[web-animations-2#trigger-behaviors]].
 
+Issue: Should 'animation-trigger-behavior' allow multiple values (currently it does)?
```

Comment:
Hm, I don't think it does?

Reply:
This syntax description from line 734 means it accepts a comma-separated list of behaviors:

 <<single-animation-trigger-behavior>>#

Reply:
Got it; I will update the PR...